src/index.html

```diff
@@ -6,9 +6,66 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="header__logo-link">
+        <img class="header__logo"
+          src="./images/logo.png"
+          alt="MOYO icon"
+        >
+      </a>
+      <nav class="header__nav nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#apple">
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#samsung">
+              Samsung
+            </a>
+          </li>
```

Comment:
Please add some line breaks for better readability:

```suggestion
    <header class="header">
    
      <a href="#home" class="header__logo-link">
        <img class="header__logo"
          src="./images/logo.png"
          alt="MOYO icon"
        >
      </a>
      
      <nav class="header__nav nav">
        <ul class="nav__list">
        
          <li class="nav__item">
            <a class="nav__link is-active" href="#apple">
              Apple
            </a>
          </li>
          
          <li class="nav__item">
            <a class="nav__link" href="#samsung">
              Samsung
            </a>
          </li>
```
And so on - for all `<li>` elements as well